src/JsonValidator/RoCrate/index.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+  import DetailsPrePanel from "../../JsonBrowser/DetailsPrePanel.svelte";
+  import { getJson } from "../../utils.js";
+  import RoCrateValidator from "./RoCrateValidator.svelte";
+
+  export let source;
+  const promise = getJson(source + "/ro-crate-metadata.json");
+</script>
+
+
+<div class="rocrate">
+<h1>Ro-crate metadata</h1>
```

Comment:
```suggestion
<h1>RO-Crate metadata</h1>
```

could potentially have a info (I) with a link to what RO-Crate is.